lib/Migration/Version5007Date20251019000000.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+declare(strict_types=1);
```

Comment:
Please increase the `version` in `appinfo/info.xml` to `5.6.0-alpha.4` (otherwise the migration will not run).